### PR TITLE
Announce RO-Crate and Galaxy export framework

### PR DESCRIPTION
**Pull request by stain**
Modified draft by @bgruening @davelopez 

Marking as "Draft" as the gif for BCO should be updated to not say "401 Not Authorized".

### CODE REVIEW

File: content/news/2023-02-06-structured-data-exports-ro-bco/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Galaxy has over the last month gained a powerful and extensible exporting framework.
+
+Two export formats have alreaady been added to Galaxy:
+
+* [BioComputeObjects](https://biocomputeobject.org/), a standard (IEEE 2791-2020) for tracking provenance information of bioinformatics pipelines for high-throughput sequencing.
+* [RO-Crate](https://www.researchobject.org/ro-crate/), a FAIR archiving format of [Research Objects](https://www.researchobject.org/) based on [schema.org](https://schema.org/) and [Bioschemas](https://bioschemas.org/).
+
+Every format can be downloaded directly, or written to pluggable Galaxy file sources (Dropbox, Google Drive, FTP, S3 ...). 
+
+The feature described here will be part of the next Galaxy release, and should become available on `usegalaxy.eu` and related instances around March 2023.
```
Comment (davelopez) [inline content/news/2023-02-06-structured-data-exports-ro-bco/index.md:22]:
```suggestion
The feature described here will be part of the next Galaxy release, and should become available on `usegalaxy.eu` and related instances around March 2023 depending on the instance configuration.
```

**Review by davelopez — COMMENTED** [review]
Thank you @stain for all the improvements!
I think the content of this post was mostly autogenerated from the PR description which was not very blog friendly :)

I've added a small clarification that the feature depends on the galaxy configuration and some instances (with the updated Galaxy version) might not have this enabled by default.

Here is also a replacement for the BCO export gif (since I cannot commit directly to the branch). It is basically the same but cutting it before the authentication error appears as I don't think is nice to submit a random test to the DB :)

send-bco-form-ui

**Comment by stain** [comment]
Thanks, @davelopez - LGTM! :-)

**Review by bgruening — APPROVED** [review]
I added a few more explicit links to the contributors. Is that ok for you @stain and @davelopez? From my side this PR is ready to go.

File: content/news/2023-02-23-structured-data-exports-ro-bco/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Adding new export plugins is relatively easy. Once the backend supports a new export format, we can add a new `InvocationExportPlugin` instance inside
+`client/src/components/Workflow/Invocation/Export/Plugins/` and register it in the *index.js* file. That's it!
+
+Each export plugin can define additional custom operations, for example to collect addional metadata from the user before the export starts.
+
+If the plugin needs to display additional components as part of the custom actions, for example for additional metadata,
+it should be done inside a modal like in the example above, and the additional component should be placed in a subfolder with the name of the plugin.
+
+This work is based on previous work from [John Chilton](https://github.com/jmchilton), who added a backend component and APIs for serving files.
+Thanks to [David](https://github.com/davelopez), [Paul](https://github.com/pauldg), [Hardley](https://github.com/HadleyKing), and [Stian](https://github.com/stian) for working on this project.
```
Comment (mvdbeek) [inline content/news/2023-02-23-structured-data-exports-ro-bco/index.md:63]:
```suggestion
Thanks to [David](https://github.com/davelopez), [Paul](https://github.com/pauldg), [Hadley](https://github.com/HadleyKing), and [Stian](https://github.com/stian) for working on this project.
```

Comment (bgruening) [reply content/news/2023-02-23-structured-data-exports-ro-bco/index.md]: Thanks!

Comment (davelopez) [reply content/news/2023-02-23-structured-data-exports-ro-bco/index.md]: I would also add thanks to John Chilton in the first place

Comment (bgruening) [reply content/news/2023-02-23-structured-data-exports-ro-bco/index.md]: Its your post :) please add or make a suggestion :)

File: content/news/2023-02-23-structured-data-exports-ro-bco/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Adding new export plugins is relatively easy. Once the backend supports a new export format, we can add a new `InvocationExportPlugin` instance inside
+`client/src/components/Workflow/Invocation/Export/Plugins/` and register it in the *index.js* file. That's it!
+
+Each export plugin can define additional custom operations, for example to collect addional metadata from the user before the export starts.
+
+If the plugin needs to display additional components as part of the custom actions, for example for additional metadata,
+it should be done inside a modal like in the example above, and the additional component should be placed in a subfolder with the name of the plugin.
+
+This work is based on previous work from [John Chilton](https://github.com/jmchilton), who added a backend component and APIs for serving files.
+Thanks to [David](https://github.com/davelopez), [Paul](https://github.com/pauldg), [Hadley](https://github.com/HadleyKing), and [Stian](https://github.com/stian) for working on this project.
```
Comment (davelopez) [inline content/news/2023-02-23-structured-data-exports-ro-bco/index.md:63]:
```suggestion
Thanks to [John Chilton](https://github.com/jmchilton), [David](https://github.com/davelopez), [Paul](https://github.com/pauldg), [Hadley](https://github.com/HadleyKing), and [Stian](https://github.com/stian) for working on this project.
```

**Comment by bgruening** [comment]
Merci!

**Pull request merged**